pkg/api/types.go

```diff
@@ -705,6 +713,8 @@ type ServiceSpec struct {
 	// not be changed by updates.
 	PortalIP string `json:"portalIP,omitempty"`
 
+	Rewrite bool `json:"rewrite,omitempty" description:"Does the load balancer rewrite packets or pass them through"`
```

Comment:
@thockin suggested changing this to a string rather than a bool to allow for a wider range of possibilities. But yes, I think it's fair to say that a cloud provider is likely to only have a single type of load balancer.